pyproject.toml

```diff
@@ -10,9 +10,10 @@ description = "ValidMind Developer Framework"
 license = "Commercial License"
 name = "validmind"
 readme = "README.pypi.md"
-version = "2.1.1"
+version = "2.1.2a0"
 
 [tool.poetry.dependencies]
+python = ">=3.8.1,<3.12"
```

Comment:
This will be put back in its place by the auto formatter 😄

Reply:
it looks like its putting it first now :shrug: ... i kind of like it better that way